spec/models/match_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe Match do
+  let(:match) do
+    build(:match, points_for_team1: 8,
+                  points_for_team2: 2)
+  end
+  let(:user) { create(:user) }
+  context "is valid" do
+    it "has team_1 points" do
+      expect(match.points_for_team1).to eq(8)
+    end
+
+    it "has team_2 points" do
+      expect(match.points_for_team2).to eq(2)
+    end
```

Comment:
most of this tests are not needed since all you need is one test in which when you have all the right attributes it is valid. The  ones that must exist are the not valid for each missing attribute